Billing.API/Services/Invoice/InvoiceService.cs

```diff
@@ -16,24 +17,41 @@ public class InvoiceService : IInvoiceService
     {
         private readonly ILogger<InvoiceService> _logger;
         private readonly IOptions<InvoiceProviderOptions> _options;
+        private readonly CryptoHelper _cryptoHelper;
 
-        public InvoiceService(ILogger<InvoiceService> logger, IOptions<InvoiceProviderOptions> options)
+        public InvoiceService(ILogger<InvoiceService> logger, CryptoHelper cryptoHelper, IOptions<InvoiceProviderOptions> options)
         {
             _logger = logger;
+            _cryptoHelper = cryptoHelper;
             _options = options;
         }
 
         public async Task<PaginatedResult<InvoiceListItem>> GetInvoices(string clientPrefix, int clientId, int page, int pageSize, string sortColumn, bool sortAsc)
         {
             var dt = await GetInvoiceRecords(clientPrefix, clientId);
 
+            var origin = clientPrefix == "CD"
+                ? "doppler"
+                : clientPrefix == "CR"
+                    ? "relay"
+                    : "clientmanager";
```

Comment:
Muy menor, pero tal vez te guste. Cuando yo me encuentro con este tipo de código, en lugar de escribirlo con una identación "jerárquica" (?), prefiero hacerlo con un formato más parecido al de _"else if"_, creo que con gran cantidad de condiciones es mucho más legible:

```suggestion
            var origin = clientPrefix == "CD" ? "doppler"
                : clientPrefix == "CR" ? "relay"
                : "clientmanager";
```

(imaginá que en lugar de dos condiciones tenés veinte 😛)